PaddleCV/rcnn/models/load_model.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
It seems that it is not very friendly to let users to write these things.

Reply:
Yes, I think so. The pretrained models are fp32, but we need fp16 weight copy to initialize fp16 weights in mixed precision. Thus I modify `load_vars` function slightly. The best way is to add this feature in Paddle. My implementation is a temporary plan for testing rcnn mixed precision training. The better implementation should be discussed.